tests/playwright-test/plugins.spec.ts

```diff
@@ -64,10 +69,15 @@ test('event order', async ({ runInlineTest }, testInfo) => {
   expect(result.passed).toBe(1);
   const logLines = await fs.promises.readFile(log, 'utf8');
   expect(logLines.split('\n')).toEqual([
+    'a configure',
+    'b configure',
     'a setup',
     'b setup',
     'globalSetup',
+    'a configure',
+    'b configure',
```

Comment:
I believe we knew this duplicate configure call would be invoked since the config is loaded first in the Runner Process and then in each Worker.

@pavelfeldman: I think we should fix the Runner/Worker so it only calls configure once and only in Runner process.

Reply:
Yes, we should fix this